### PR TITLE
Add floor height for peer rigs.

### PR DESCRIPTION
**Pull request by ChrisEddy**

### CODE REVIEW

File: avatars/avatars.js [hunk 1]
```diff
@@ -1346,6 +1346,10 @@ class Avatar {
     this.poseManager.vrTransforms.floorHeight = floorHeight;
   }
 
+  getFloorHeight() {
+    return(this.poseManager.vrTransforms.floorHeight);
```
Comment (avaer) [inline avatars/avatars.js:1350]: Weird parens.

File: index.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ https.createServer({
   cert: CERT,
   key: PRIVKEY,
 }, app)
-  .listen(4443);
+  .listen(3001);
```
Comment (avaer) [inline index.js:22]: Not related to this PR, but still good.

File: planet.js [hunk 3]
```diff
@@ -739,35 +739,36 @@ const _connectRoom = async roomName => {
...
         if (method === 'pose') {
           const {pose} = j;
-          const [head, leftGamepad, rightGamepad] = pose;
+          const [head, leftGamepad, rightGamepad, floorHeight] = pose;
 
-          console.log('got pose', head[0].join(','));
+          // console.log('got pose', head[0].join(','));
           window.remotePose = pose;
-          window.remoteRig = playerRig;
+          window.remoteRig = peerRig;
```
Comment (avaer) [inline planet.js:746]: Probably shouldn't latch to window anymore; this was for debugging.

**Review by avaer — APPROVED** [review]

File: planet.js (hunk 3 above)
Comment (ChrisEddy) [reply planet.js]: fixed

File: avatars/avatars.js (hunk 1 above)
Comment (ChrisEddy) [reply avatars/avatars.js]: fixed

**Pull request merged**